docs/reference/components/transforms/regex_parser.cue

```diff
@@ -118,4 +125,24 @@ components: transforms: regex_parser: {
       }
     }
   ]
+  how_it_works: {
+    failed_parsing: {
+      title: "Failed Parsing"
+      body: #"""
+            By default, if the input message text does not match any of the configured regular expression patterns, this transform will log an error message but leave the log event unchanged. If you instead wish to have this transform drop the event, set `drop_failed = true`.
+            """#
+    }
+    regex_debugger: {
+      title: "Regex Debugger"
+      body: #"""
+            If you are having difficulty with your regular expression not matching text, you may try debugging your patterns at [Regex 101](https://regex101.com/). This site includes a regular expression tester and debugger. The regular expression engine used by Vector is most similar to the "Go" implementation, so make sure that is selected in the "Flavor" menu.
```

Comment:
@bruceg you can use the definitions in https://github.com/timberio/vector/blob/master/docs/reference/urls.cue for links.